Singularity/Singularity/Screen/ELogEventType.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Runtime.Serialization;
+
+namespace Singularity.Screen
+{
+    public enum ELogEventType
+    {
+        [EnumMember(Value = "Unit attacked")]
+        UnitAttacked,
+
+        [EnumMember(Value = "Platform attacked")]
+        PlatformAttacked,
+
+        [EnumMember(Value = "Platform built")]
+        PlatformBuilt,
```

Comment:
Is that triggered if a Platform changes from Blueprint to nonBlueprint? if not, please do so :)